test/support/api.ts

```diff
@@ -11,10 +11,18 @@ export class Api {
   constructor(private readonly app: App) {}
 
   get(url: string): request.Test {
+    const apiKey = process.env.API_KEY;
+    const strategy = process.env.API_KEY_STRATEGY;
```

Comment:
Can we get these from the `ENVIRONMENT_VARIABLES` constant please?

Reply:
updated